Sources/Purchasing/Offering.swift

```diff
@@ -165,6 +173,17 @@ import Foundation
 
 }
 
+extension Offering {
+
+    /**
+     Gets a value from `metadata` and falls back to a specified default value.
+     */
+    public func getMetadataValue<T>(for key: String, defaultValue: T) -> T {
```

Comment:
Being nit-picky since this is `public` so we want to get it right from the beginning.
Maybe:
```suggestion
    public func getMetadataValue<T>(for key: String, default: T) -> T {
```
Swift guidelines require avoiding that alliteration.